.changeset/hot-schools-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Migrate the Popover component from popper to floating-ui. Popper modifiers are longer supported, while offsetProp is exposed to allow for flexible placement of floating element.
```

Comment:
I believe you have a small typo here, popper modifiers are no longer supported.
Maybe you can rephrase it 
'Migrated the Popover component from Popper to Floating UI. Popper modifiers are no longer supported, use offset prop for flexible placement of floating element instead.

Reply:
You're right! @amelako. The suggested wording is more correct. I committed the change. Thanks! :)